library/alloc/src/vec/source_iter_marker.rs

```diff
@@ -1,3 +1,106 @@
+///! Inplace iterate-and-collect specialization for `Vec`
```

Comment:
Aren't these supposed to be `//!` comments?

Reply:
🤦‍♂️ And there I was wondering why my IDE was confused again and wouldn't render the comment. Fixed.